includes/TinyMCE.php

```diff
@@ -89,20 +91,26 @@ public function web_stories_mce_plugin( array $plugins ) {
 	}
 
 	/**
-	 * Enqueue scripts related to tinymce button for web stories.
+	 * Enqueue related scripts.
 	 *
 	 * @since 1.5.0
 	 *
 	 * @return void
 	 */
-	public function web_stories_tinymce_scripts() {
+	public function enqueue_assets() {
 		$this->enqueue_style( 'wp-components' );
 
-		// Can't use enqueue_script() because the script needs to be loaded via the mce_external_plugins filter.
+		// Can't use Assets::enqueue_script() because the script needs to be loaded via the mce_external_plugins filter.
 		$asset = $this->get_asset_metadata( 'tinymce-button' );
+
 		foreach ( $asset['dependencies'] as $script ) {
 			wp_enqueue_script( $script );
 		}
```

Comment:
```suggestion
```

This is not needed because they are dependencies of `web-stories-tinymce-button`

Reply:
Why was the marked as resolved?